### PR TITLE
Feat ticket custom form:Added ticket addtion deletion and editing form

### PR DESCRIPTION
**Pull request by Amartyakumar**
Added the add ticket form
Edit ticket form
delete ticket

### CODE REVIEW

File: config/templates/grids/eventTickets-grid/actions/grid/new-item.vue [hunk 1]
```diff
@@ -0,0 +1,480 @@
...
+import registrationtype from '~/config/apps/event/gql/registrationType.gql'
+import event from '~/config/apps/event/gql/event.gql'
+import { formatGQLResult } from '~/utility/gql.js'
+import { required } from '~/utility/rules.js'
+import { getApiUrl } from '~/utility'
+export default {
+  props: {
+    refresh: {
+      type: Function,
+      required: false,
```
Comment (SukantGujar) [inline config/templates/grids/eventTickets-grid/actions/grid/new-item.vue:182]: No need for required:false, it is default.

File: config/templates/grids/eventTickets-grid/actions/grid/new-item.vue [hunk 2]
```diff
@@ -0,0 +1,480 @@
...
+      default: () => false,
```
Comment (SukantGujar) [inline config/templates/grids/eventTickets-grid/actions/grid/new-item.vue:183]: Just default: false is enough.

File: config/templates/grids/eventTickets-grid/actions/grid/new-item.vue [hunk 3]
```diff
@@ -0,0 +1,480 @@
...
+        Status: '',
+      },
+      eventData: {},
+      setAttendeeType: {},
+      Symbol: '',
+    }
+  },
+  computed: {},
+  mounted() {
+    this.getDropDownData('TicketType')
```
Comment (SukantGujar) [inline config/templates/grids/eventTickets-grid/actions/grid/new-item.vue:221]: No promises please. We have discussed this so often.

**Review by SukantGujar — CHANGES_REQUESTED** [review]

File: config/templates/grids/eventTickets-grid/actions/grid/new-item.vue (hunk 2 above)
Comment (Amartyakumar) [reply config/templates/grids/eventTickets-grid/actions/grid/new-item.vue:182]: default:false was giving linterror so returned as function with default()=> false. Resolved required part

File: config/templates/grids/eventTickets-grid/actions/grid/new-item.vue (hunk 3 above)
Comment (Amartyakumar) [reply config/templates/grids/eventTickets-grid/actions/grid/new-item.vue]: resolved

**Review by SukantGujar — APPROVED** [review]

**Pull request merged**